cfpb/consumerfinance.gov
# Add TOC Expandable For Mobile Users/Small Screens

Using [CFPB's design system](https://cfpb.github.io/design-system), added in an [expandable](https://cfpb.github.io/design-system/components/expandables) table of contents that fit the design patterns put forward by the previous reports-as-webages team.  

Currently, the expandable does not open or close, and lacks the basic functionality that it comes with.  @contolini has agreed to take a look and see if he can discover the issue. 

---




## Additions

- added the expandable via javascript

## How to test this PR

1. clone the branch and run the build locally 
2. create a new FIG content page in wagtail, adding in several sections/sub-sections
3. shrink the browser window in wagtail preview to see the expandable TOC


## Screenshots


## Notes and todos
- need to work on getting svg files added to the expandable

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
